src/routes/plans/+page.svelte

```diff
@@ -532,170 +626,268 @@
+                  disabled
+                  class="st-input w-100"
+                  name="duration"
+                  value={convertUsToDurationString(getIntervalInMs(selectedPlan.duration) * 1000)}
+                />
+              </Input>
+              <Input layout="inline">
+                <label class="plan-metadata-item-label" for="tags">Tags</label>
+                <TagsInput
+                  use={[
+                    [
+                      permissionHandler,
+                      {
+                        hasPermission: canCreate,
+                        permissionError,
+                      },
+                    ],
+                  ]}
+                  disabled
+                  options={$tags}
+                  selected={selectedPlan.tags.map(({ tag }) => tag)}
+                  on:change={onTagsInputChange}
+                />
+              </Input>
+            </fieldset>
+          </div>
           <fieldset>
-            <label for="plan-duration">Plan Duration</label>
-            <input class="st-input w-100" disabled id="plan-duration" name="duration" value={durationString} />
+            <button class="st-button w-100" on:click={showPlan}>Open plan</button>
           </fieldset>
-
-          <Field field={simTemplateField}>
-            <label for="simulation-templates" slot="label"> Simulation Templates </label>
-            <select
-              class="st-select w-100"
-              data-type="number"
-              disabled={!$simulationTemplates.length}
-              name="simulation-templates"
-              use:permissionHandler={{
-                hasPermission: canCreate,
-                permissionError,
-              }}
-            >
-              {#if !$simulationTemplates.length}
-                <option value="null">Empty</option>
-              {:else}
-                <option value="null" />
-                {#each $simulationTemplates as template}
-                  <option value={template.id}>
-                    {template.description}
+        {:else}
+          <form on:submit|preventDefault={createPlan}>
+            <AlertError class="m-2" error={$createPlanError} />
+
+            {#if isPlanImportMode}
+              <fieldset>
+                <label for="file">Plan File (JSON)</label>
+                <div class="import-input-container">
+                  <input
+                    class="w-100"
+                    name="file"
+                    type="file"
+                    accept="application/json"
+                    bind:files={planUploadFiles}
+                    bind:this={planUploadFileInput}
+                    use:permissionHandler={{
+                      hasPermission: canCreate,
+                      permissionError,
+                    }}
+                    on:change={onPlanFileChange}
+                  />
+                  {#if planUploadFiles}<button class="st-button secondary" type="button" on:click={hideImportPlan}>
```

Comment:
Needs alignment with the native button 
<img width="273" alt="image" src="https://github.com/user-attachments/assets/a6a45460-8e7c-42d9-8c69-93979b9be0b1">

Reply:
Thoughts on making the "clear" button say "close" if no files are selected? Kind of wanting some way to get rid of the import in case I didn't actually want to import. Wasn't super intuitive to me to click import again to get rid of the file uploader and I wasn't sure if I needed to upload a file now that it was present or if it was actually optional.

Reply:
The `Import` toggle wasn't really part of the design. I just threw that in there. @lklyne do you have any thoughts on this?

Reply:
Updated the form design thanks to @lklyne's feedback!
![Screenshot 2024-08-06 at 4 29 21 PM](https://github.com/user-attachments/assets/53d38806-941f-4845-a884-ae6b987e4a18)